### PR TITLE
Listing of Props in README to being basis of documentation

### PR DESCRIPTION
**Pull request by jmfurlott**
_An incomplete PR_

Just wanted to see if this was right direction -- I listed the props based on the `Autocomplete.js` file and wrote their PropTypes and a little bit about their functionality.  Is this what you were thinking about documentation? Wanted to ask before we go down the wrong path.

See https://github.com/reactjs/react-autocomplete/issues/148

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,45 @@ Write them:
...
+but let the user find values based solely on a specifically key.
+
+### `renderItem` (function)
+
+A function that returns a React element to show as an item in the list. Two
+props are passed to the function, `item` and `isHighlighted`, which allow the
+items in the list to be customized and styled. To style the container of these
+items, review the `menuStyle` prop.
+
+## Optional props
```
Comment (sprjr) [inline README.md]: @jmfurlott Required props looks great! maybe the additional props here would be better formatted with a simple table? [react-select](https://github.com/JedWatson/react-select#further-options) has a good example I think.

Comment (jmfurlott) [inline README.md]: Great idea and example.  Will have that for additional props next!

Comment (sprjr) [inline README.md]: Sure thing. Thanks for the help! Let me know when you think this PR is in a good spot and we can get it merged in.

**Comment by CMTegner** [comment]
We now have documentation the the README. I'm sure this can be improved upon if you're still keen on working on the documentation.

**Pull request closed without merging**